contrib/helm/harbor/templates/adminserver/adminserver-cm.yaml

```diff
@@ -54,6 +54,7 @@ data:
   CLAIR_DB_PORT: "5432"
   CLAIR_DB: "{{ .Values.clair.postgresDatabase }}"
   CLAIR_DB_USERNAME: "{{ .Values.clair.postgresUser }}"
+  CLAIR_URL: "http://clair:6060"
```

Comment:
It should be `http://{{ template "harbor.fullname" . }}-clair:6060`

Reply:
It's not:
https://github.com/vmware/harbor/blob/master/contrib/helm/harbor/templates/clair/clair-svc.yaml#L8

We'll fix this in further refactory to make things more consistent.